### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by stephan281094**
This pull request adds the following:
* Begin of profile page
* Login functionality with Github
* Logout functionality
* Improved header
* Improved design (cleaner colors)

### CODE REVIEW

File: client/templates/common/layout.html [hunk 1]
```diff
@@ -1,5 +1,12 @@
+<head>
+	<meta charset="utf-8">
+	<title>Code Glimpse</title>
+	<meta name="description" content="Code Glimpse is a social network created by developers for developers.">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+</head>
+
 <template name="layout">
-	{{> metadata }}
+	{{> login}}
```
Comment (stephan281094) [inline client/templates/common/layout.html]: This should be removed.

File: client/templates/common/login.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template name="login">
+	{{ #if login }}
+		<div class="modal">
+
+		</div>
+	{{ /if }}
+</template>
```
Comment (stephan281094) [inline client/templates/common/login.html]: This file should be removed.

File: config/routes.coffee [hunk 3]
```diff
@@ -5,6 +5,10 @@ Router.configure
 
 Router.route '/', ->
 	@render 'home'
+	Session.set('login', false)
```
Comment (stephan281094) [inline config/routes.coffee]: Not used.

File: config/routes.coffee [hunk 4]
```diff
@@ -5,6 +5,10 @@ Router.configure
...
 
 Router.route '/login', ->
 	@render 'home'
```
Comment (stephan281094) [inline config/routes.coffee]: Not used.

File: client/controllers/home.coffee [hunk 5]
```diff
@@ -0,0 +1,4 @@
+if Meteor.isClient
+	Template.login.helpers
+		login: ->
+			Session.get 'login'
```
Comment (stephan281094) [inline client/controllers/home.coffee]: Unused.

**Comment by AwesomeRoy** [comment]
Login functionality with Github, Nice mate! All looks Awesome :)

**Pull request merged**